connect/runtime/src/main/java/org/apache/kafka/connect/cli/ConnectStandalone.java

```diff
@@ -60,11 +60,15 @@ public static void main(String[] args) throws Exception {
             Exit.exit(1);
         }
 
+        Time time = Time.SYSTEM;
+        log.info("Kafka Connect standalone worker initializing ...");
+        long initStart = time.hiResClockMs();
+
         String workerPropsFile = args[0];
         Map<String, String> workerProps = !workerPropsFile.isEmpty() ?
                 Utils.propsToStringMap(Utils.loadProps(workerPropsFile)) : Collections.<String, String>emptyMap();
 
-        Time time = Time.SYSTEM;
+        log.info("Scanning for plugin classes. This might take a moment ...");
         Plugins plugins = new Plugins(workerProps);
```

Comment:
What about logging that it finished scanning the plugin directories and how long that took, especially since that requires the most time?

Reply:
I think it suffices to bundle everything as initialization phase. And, it's true. Initialization is actually plugin scanning. I added a message to indicate when it starts, but getting overly specific now doesn't seem all that necessary. That's what I thought.